samples/suit/ab/README.rst

```diff
@@ -32,30 +33,279 @@ To perform a SUIT update over UART using the SMP protocol, install SUIT commands
+
+The suit-validate sequence
+--------------------------
+
+The suit-validate sequence of the root manifest is used to select the valid set of manifests to boot from.
+The result of this check is written into the ``BOOT_STATUS`` manifest variable.
+The following activity diagram presents the logic, implemented inside the suit-validate sequence.
+
+.. figure:: img/ab_root_validate.png
+
+   :alt: The suit-validate sequence of the root manifest
+
+The suit-invoke sequence
+------------------------
+
+The suit-invoke sequence of the root manifest reads the ``BOOT_STATUS`` manifest variable value and act upon it.
+If the application has been just updated, which is indicated by the ``CONFIRM_SET_A`` or ``CONFIRM_SET_B`` manifest variable set to ``JUST_INSTALLED``,
+and correctly validated by the suit-validate sequence, the confirmation variable is reset to ``NOT_CONFIRMED``.
+It is now the applications responsibility to set this variable to ``CONFIRMED``.
+Otherwise this variant will be considered as degraded on the next boot attempt.
+The following activity diagram presents the logic, implemented inside the suit-invoke sequence.
+
+.. figure:: img/ab_root_invoke.png
+
+   :alt: The suit-invoke sequence of the root manifest
+
+It is worth noticing that apart from regular application/radio/root manifest, the application recovery manifest is also defined.
+In the A/B sample there is no separate recovery binary though.
+The recovery manifest is very similar to the root manifest instead, so in case of the root manifest damage, the recovery manifest serves as a backup logic.
 
 User interface
 **************
 
 LED 0:
+    This LED is controlled by the application variant ``A``.
     The ``CONFIG_N_BLINKS`` Kconfig option sets the number of blinks.
-    The default is set to ``1``, causing a single blink to indicate *Version 1*.
+    This time the default LED state is ``On``, so it is clearly visible that the application has booted.
+    The default of the Kconfig is set to ``1``, causing a single blink to indicate *Version 1*.
     This value can be incremented to represent an update, such as setting it to ``2`` to indicate *Version 2*.
+    Using the ``-Dab_CONFIG_N_BLINKS`` command line option, the number of blinks can be set for both variants of the application.
+
+LED 1:
+    This LED is controlled by the application variant ``B``.
+    Using the ``-Dab_slot_b_CONFIG_N_BLINKS`` command line option, it is possible to change the number of blinks just for the application variant ``B``.
+
+LED 2 and 3:
+    These LEDs indicate the status of manifests inside the device:
+
+    * If both LEDs are turned ``On``, the application is booted normally.
+    * If both LEDs blink simultaneously, the application is booted in a degraded mode (i.e. into an unconfirmed variant).
+    * If both LEDs blink out of phase, the application is booted using the recovery manifest.
+    * If only LED 2 blinks, the application is booted without radio capabilities.
+
+BUTTON 3:
```

Comment:
BUTTON 4, not 3: https://github.com/robertstypa/sdk-nrf/blob/f9eff8c312c3d04dfc33cd4378d8746ff9161157/samples/suit/ab/src/main.c#L25

Reply:
To my understanding - documentation counts LEDs and BUTTONs from 0...